### ISSUE TITLE
Support for offline mode is required for HPC

### ISSUE THREAD

**Issue by alahiff** [simvue-io/integrations#45]
## Problem Statement
The connectors don't appear to work for offline mode. For FDS I get:
```
RuntimeError: Retrieval of current tags is not implemented for offline running
```

## Solution
Support for offline mode is essential for HPC

## Alternatives Considered
The alternative is to use a custom script rather than an official connector.

## Additional Context
None.

**Comment by alahiff** [comment]
Actually, maybe offline mode is supported, but e.g. in the FDS container there is:
```
    run.init(f"fds_simulation_no_vents")
    run.update_tags(["fds", "no_vents"])
```
The `update_tags` is what's causing the problem. Maybe this is an issue that needs to be fixed in the Python API rather than here.

**Comment by wk9874** [comment]
Yep this will be a python-api issue, and has probably changed relatively recently with changes to `update_tags` working to add to existing tags, rather than overwrite them. So `update_tags` now requires existing tags to be retrieved first using `list_tags`, but this returns your error above if tried in offline mode (in `simvue/factory/proxy/offline.py`)

For now you can remove any calls to the `update_tags` method from the python script in the docker container and see if it works? Should hopefully work, since the connector itself doesn't use `update_tags` anywhere

I've raised an issue on the python-api repo: https://github.com/simvue-io/python-api/issues/629

**Comment by wk9874** [comment]
This does make me realise that there are no tests for offline mode in the integrations repo though... will raise a separate issue for that :)
